packages/trader/src/Modules/Trading/Components/Elements/mobile-widget.jsx

```diff
@@ -97,22 +82,15 @@ const MobileWidget = observer(({ toggleDigitsWidget, is_collapsed }) => {
                     <div className='mobile-widget__duration' onClick={() => toggleWidget(0)}>
                         {getHumanReadableDuration()}
                     </div>
-                    <div className='mobile-widget__amount--container' onClick={() => toggleWidget(1)}>
-                        <div className='mobile-widget__amount'>
-                            <Money amount={amount} currency={currency} show_currency />
-                        </div>
-                        <div className='mobile-widget__type'>{stakeOrPayout()}</div>
+                    <div className='mobile-widget__amount' onClick={() => toggleWidget(1)}>
+                        <Money amount={amount} currency={currency} show_currency />
                     </div>
-                </div>
-            )}
-            <TradeParamsModal is_open={is_open} toggleModal={() => toggleWidget(tab_index)} tab_index={tab_index} />
-            {isVisible('last_digit') && is_collapsed && (
-                <div className='mobile-widget' onClick={toggleDigitsWidget}>
-                    <div className='mobile-widget__amount'>
-                        <Localize i18n_default_text='Digit: {{last_digit}} ' values={{ last_digit }} />
+                    <div className='mobile-widget__type' onClick={() => toggleWidget(1)}>
```

Comment:
Previously `mobile-widget__amount` (stake amount) and `mobile-widget__type` ('Stake' or 'Payout') were in one container on which `toggleWidget(1)` was put onClick. That 'uniting' in one container broke style as there was not enough space for big stake (more then 10USD). In new implementation, each of 3 parts (`mobile-widget__duration`, `mobile-widget__amount` and `mobile-widget__type`) has its own `toggleWidget` with different arguments and the clickable area is expanded with the usage of CSS.